src/ORM/Behavior/CounterCacheBehavior.php

```diff
@@ -236,7 +241,10 @@ protected function _processAssociation(Event $event, EntityInterface $entity, As
             $assoc->getTarget()->updateAll([$field => $count], $updateConditions);
 
             if (isset($updateOriginalConditions)) {
-                if (!is_string($config) && is_callable($config)) {
+                if (is_callable($config)) {
+                    if (is_string($config)) {
+                        throw new RuntimeException('You must not use a string as callable.');
+                    }
```

Comment:
This is a behavior change in that a callable string will now raise an exception. Previously it would be ignored and the default implementation of `_getCount()` would be used. Not sure this will be relevant though.

Reply:
that method has a array typehint. afaik that would blow up either way.